src/main/drivers/light_ws2811strip.c

```diff
@@ -179,6 +181,7 @@ void ws2811UpdateStrip(ledStripFormatRGB_e ledFormat)
 {
     // don't wait - risk of infinite block, just get an update next time round
     if (!ws2811Initialised || ws2811LedDataTransferInProgress) {
+        ignoreTaskStateTime();
```

Comment:
How is this related to the contents of this pull request?

Reply:
This relates to the scheduling which is part of this PR.